### ISSUE TITLE
A11y_30days_Welcome to 30 Days of Learning_Resize: When resize testing is performed in "Menu hamburger icon" , the Close button is only announcing as "BUTTON" using Narrator Screen Reader.

### ISSUE THREAD

**Issue by Ashutoshsahoo164** [microsoft/30daysof#180]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### GitHub Tags:
#A11yTCS;#A11yMAS;#A11ySev2;#30days;#P3_30days_Dec22;#A11yWCAG2.1;#WCAG1.4.4;#WCAG1.4.10;#DesktopWeb;#Win11;#ChromiumEdge

### Environment Details:
Application Name: 30days
#URL: https://microsoft.github.io/30daysof/
Windows Version: Win11
Chromium Edge Version: 110.0.1556.0 (Official build) dev (64-bit)

### Pre-Requisite:
To apply Resize resolution, follow the below steps:

Zoom the browser screen to 200%
Refresh the page.

### Repro Steps:
1. Hit the URL: https://microsoft.github.io/30daysof/
2. Zoom the page to 200% , turn on Narrator .
3. Tab till "Close button" present in Hamburger Menu.
4. Observe that Narrator is announcing "Button" when the focus comes to Close button.
### Actual:
While Navigating the screen with Narrator screen reader, Narrator is announcing "Button" when focus lands on Close button.

**Note** – This issue is reproducing with NVDA screen reader.
Also this issue is reproducing with Reflow Resolution.

### Expected:
Narrator and NVDA Screen reader should announce "Close Button" only.

### User Impact:
Screen reader user will be confused when they will navigate to this close button.

### Attachment:
Narrator Attachment:
![Narrtor Close button](https://user-images.githubusercontent.com/85863895/209615416-5bfc5cb1-5812-48a5-a97c-2b5e46ea707c.gif)

NVDA Attachment:
![Screenshot (42)](https://user-images.githubusercontent.com/85863895/209615631-45bfbe78-b385-43f8-8227-1c2f3af77be6.png)

**Comment by nitya** [comment]
@Ashutoshsahoo164 - I seem to be missing the narrator attachment (audio walkthrough)? I see only screenshots. Site has been updated for other a11y fixes. Can you verify if this issue is still present and provide recording?

**Comment by Ashutoshsahoo164** [comment]
![Screenshot (15)](https://user-images.githubusercontent.com/85863895/218804723-a0e82a76-e97a-4a6c-94bf-042f052fa03c.png)

Hi @nitya Please check the screenshot with Narrator buddy speech viewer

**Comment by nitya** [comment]
@Ashutoshsahoo164 we have decided to unpublish the site and deactivate it from internal listings. Can you verify and close this issue now that website is no longer active?

**Comment by Ashutoshsahoo164** [comment]
As per the above comments , closing this bug.
